src/issuingPurchase/issuingPurchase.php

```diff
@@ -15,32 +15,33 @@ class IssuingPurchase extends Resource
-        - merchantName [string, default null]: merchant name. ex: "Google Cloud Platform"
-        - walletId [string, default null]: virtual wallet ID. ex: "5656565656565656"
-        - methodCode [string, default null]: method code. ex: "chip", "token", "server", "manual", "magstripe" or "contactless"
-        - score [float, default 0.0]: internal score calculated for the authenticity of the purchase. ex: 7.6
-        - issuingTransactionIds [string, default null]: ledger transaction ids linked to this Purchase
-        - endToEndId [string, default null]: central bank's unique transaction ID. ex: "E79457883202101262140HHX553UPqeq"
-        - status [string, default null]: current IssuingCard status. ex: "approved", "canceled", "denied", "confirmed" or "voided"
-        - tags [string, default null]: list of strings for tagging. ex: ["travel", "food"]
-        - created [string, default null]: creation datetime for the IssuingPurchase. ex: "2020-03-10 10:30:00.000"
-        - updated [string, default null]: latest update datetime for the IssuingPurchase. ex: "2020-03-10 10:30:00.000"
+        - cardId [string]: unique id returned when IssuingCard is created. ex: "5656565656565656"
+        - cardEnding [string]: last 4 digits of the card number. ex: "1234"
+        - amount [integer]: IssuingPurchase value in cents. Minimum = 0 (any value will be accepted). ex: 1234 (= R$ 12.34)
+        - tax [integer]: IOF amount taxed for international purchases. ex: 1234 (= R$ 12.34)
+        - issuerAmount [integer]: issuer amount. ex: 1234 (= R$ 12.34)
+        - issuerCurrencyCode [string]: issuer currency code. ex: "USD"
+        - issuerCurrencySymbol [string]: issuer currency symbol. ex: "$"
+        - merchantAmount [integer]: merchant amount. ex: 1234 (= R$ 12.34)
+        - merchantCurrencyCode [string]: merchant currency code. ex: "USD"
+        - merchantCurrencySymbol [string]: merchant currency symbol. ex: "$"
+        - merchantCategoryCode [string]: merchant category code. ex: "eatingPlacesRestaurants"
+        - merchantCountryCode [string]: merchant country code. ex: "USA"
+        - merchantFee [string]: fee charged by the merchant to cover specific costs, such as ATM withdrawal logistics, etc. ex: 200 (= R$ 2.00)
+        - acquirerId [string]: acquirer ID. ex: "5656565656565656"
+        - merchantId [string]: merchant ID. ex: "5656565656565656"
+        - merchantName [string]: merchant name. ex: "Google Cloud Platform"
+        - walletId [string]: virtual wallet ID. ex: "5656565656565656"
+        - methodCode [string]: method code. ex: "chip", "token", "server", "manual", "magstripe" or "contactless"
+        - score [float]: internal score calculated for the authenticity of the purchase. ex: 7.6
+        - issuingTransactionIds [string]: ledger transaction ids linked to this Purchase
+        - endToEndId [string]: central bank's unique transaction ID. ex: "E79457883202101262140HHX553UPqeq"
```

Comment:
Isso aqui ta com a descrição errada e com o exemplo errado:

Unique id used to identify the transaction through all of its life cycle, even before the purchase is denied or accepted and gets its usual id. ex: endToEndId="679cd385-642b-49d0-96b7-89491e1249a5"